bundles/org.openhab.binding.boschshc/src/main/java/org/openhab/binding/boschshc/internal/devices/smokedetector/SmokeDetector2Handler.java

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright (c) 2010-2024 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.boschshc.internal.devices.smokedetector;
+
+import static org.openhab.binding.boschshc.internal.devices.BoschSHCBindingConstants.CHANNEL_SIGNAL_STRENGTH;
+
+import java.util.List;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.openhab.binding.boschshc.internal.devices.AbstractSmokeDetectorHandler;
+import org.openhab.binding.boschshc.internal.exceptions.BoschSHCException;
+import org.openhab.binding.boschshc.internal.services.communicationquality.CommunicationQualityService;
+import org.openhab.binding.boschshc.internal.services.communicationquality.dto.CommunicationQualityServiceState;
+import org.openhab.core.thing.Thing;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The smoke detector 2 warns you in case of fire.
+ *
+ * @author Patrick Gell - Initial contribution
+ */
+@NonNullByDefault
+public class SmokeDetector2Handler extends AbstractSmokeDetectorHandler {
+
+    private final Logger logger = LoggerFactory.getLogger(getClass());
+
+    public SmokeDetector2Handler(Thing thing) {
+        super(thing);
+    }
+
+    @Override
+    protected void initializeServices() throws BoschSHCException {
+        super.initializeServices();
+
+        this.createService(CommunicationQualityService::new, this::updateChannels, List.of(CHANNEL_SIGNAL_STRENGTH),
+                true);
+    }
+
+    private void updateChannels(CommunicationQualityServiceState communicationQualityServiceState) {
+        logger.debug("***** update communication quality with {}", communicationQualityServiceState.quality);
```

Comment:
Is this log really needed?
If so, maybe we should remove the asterisks and Capitalize all log messages: `Updating communication quality for Smoke Detector II with value {}`

Reply:
upps, that was just a debug while I did some troubleshooting and used the asterisks to find it faster in the log 😬 

will remove the whole log

Reply:
log has been removed